node/src/BaseClient.ts

```diff
@@ -2203,6 +2204,29 @@ export class BaseClient {
         return this.createWritePromise(createBrpop(keys, timeout));
     }
 
+    /** Adds all elements to the HyperLogLog data structure stored at the specified `key`.
+     * Creates a new structure if the `key` does not exist.
+     * When no elements are provided, and `key` exists and is a HyperLogLog, then no operation is performed.
+     * If `key` does not exist, then the HyperLogLog structure is created.
```

Comment:
this line and the one two lines above say the same thing, we can probably remove one of them. Same for the doc in Transaction.ts

Reply:
Updated in python client too